clients/feeder/transaction.go

```diff
@@ -44,22 +45,72 @@ func (fs *FinalityStatus) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+type TransactionType uint8
+
+const (
+	Invalid TransactionType = iota
```

Comment:
Why not `iota+1` on the next line? I remember discussing this but forget what the trade-off is.

Reply:
I guess because untyped constant will be converted to Invalid automatically. Like this:
```
...
    return 0, err
```
And later on we can safely compare this returned value with `Invalid` const if we need to

Reply:
> we can safely compare this returned value with Invalid const if we need to

Pretty much